src/elements/content-sidebar/messages.js

```diff
@@ -117,6 +117,11 @@ const messages = defineMessages({
         defaultMessage: 'Modify General Task',
         description: 'modal title for when editing an existing general task',
     },
+    expandBoxAI: {
+        id: 'be.contentSidebarExpandBoxAI',
+        description: 'Default message for Box AI expand button in sidebar header',
+        defaultMessage: 'Expand',
```

Comment:
Minor, but let's move this up a line to stay consistent with the other messages.